test/integration_test/physical_machine/run.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 Chaos Mesh Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $cur
+
+echo "download and deploy chaosd"
+
+# TODO: use a released version
+curl -fsSL -o chaosd-platform-linux-amd64.tar.gz https://mirrors.chaos-mesh.org/chaosd-platform-linux-amd64.tar.gz
```

Comment:
I find this mirror not the latest release version. I think we may make it update.

Reply:
Rest LGTM.

Reply:
Yes, since that the latest release v1.0.1 has something different with this pr (for example, I remove the `worker` config for memory stress), so I use an unstable release file here. I add a TODO comment above, will update it when the new version is released.